courses/fundamentals_of_ada/quiz/private_incomplete/quiz.rst

```diff
@@ -0,0 +1,12 @@
+.. code:: Ada
```

Comment:
How is this different from privacy completion?

Reply:
An incomplete type can be completed by a limited, tagged, or constrained type, whereas the private example cannot. Modifying the quiz to make the distinction clearer